views/BD103.html

```diff
@@ -4,13 +4,39 @@
 <head>
 	<meta charset="UTF-8">
 	<meta name="viewport" content="width=device-width, initial-scale=1.0">
-	<meta http-equiv="refresh" content="3; URL=https://bd103.github.io">
-	<title>BD103 Woo</title>
+	<title>BD103</title>
+  <style>
+    @import url('https://fonts.googleapis.com/css2?family=Montserrat&family=Pangolin&display=swap');
+    body {
+      font-family: "Montserrat", sans-serif;
+      font-size: 20px;
+      background: linear-gradient(to bottom right, darkcyan, lightcyan);
+      background-repeat: no-repeat;
+      background-size: cover;
+      background-attachment: fixed;
+    }
+    h1, h2 {font-family: "Pangolin", cursive;}
+    .title {font-size: 60px;}
+    .center {text-align: center;}
+    .left {text-align: left;}
+    .right {text-align: right;}
+  </style>
```

Comment:
Please make this a but tidier by adding two indentations for each css element like this:

``` css
body {
      font-family: "Montserrat", sans-serif;
      font-size: 20px;
      background: linear-gradient(to bottom right, darkcyan, lightcyan);
      background-repeat: no-repeat;
      background-size: cover;
      background-attachment: fixed;
    }
    h1, h2 {font-family: "Pangolin", cursive;}
        .title {font-size: 60px;}
        .center {text-align: center;}
        .left {text-align: left;}
        .right {text-align: right;}
  
```
Please update that.